Lib/ensurepip/__init__.py

```diff
@@ -1,8 +1,10 @@
 import os
 import os.path
-import pkgutil
 import sys
 import tempfile
+from . import _bundled
```

Comment:
[PEP 8 says](https://www.python.org/dev/peps/pep-0008/#imports) to sort imports so that stdlib imports com first, then a blank line, then package-local modules.